### PR TITLE
fixes: #846 #847: new default for SQS fetch message count. 

### PR DESCRIPTION
**Pull request by SimY4**
- new default for SQS fetch message count
- SqsConfig moved to another package.

### CODE REVIEW

File: fs2-aws-sqs/src/main/scala/fs2/aws/sqs/SqsConfig.scala [hunk 1]
```diff
@@ -1,9 +1,9 @@
-package sqs
+package fs2.aws.sqs
 
 import scala.concurrent.duration.*
 
 case class SqsConfig(
     queueUrl: String,
     pollRate: FiniteDuration = 3.seconds,
-    fetchMessageCount: Int = 100
+    fetchMessageCount: Int = 10
```
Comment (semenodm) [inline fs2-aws-sqs/src/main/scala/fs2/aws/sqs/SqsConfig.scala:8]: can you put link in the comment to the aws documentation with the sqs limits?

Comment (SimY4) [reply fs2-aws-sqs/src/main/scala/fs2/aws/sqs/SqsConfig.scala:8]: https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_ReceiveMessage.html

Quote:

> Retrieves one or more messages (up to 10), from the specified queue.

Comment (semenodm) [reply fs2-aws-sqs/src/main/scala/fs2/aws/sqs/SqsConfig.scala:8]: sorry i mean into the code comment.

Comment (SimY4) [reply fs2-aws-sqs/src/main/scala/fs2/aws/sqs/SqsConfig.scala:11]: @semenodm Ah. Got it done.

**Comment by semenodm** [comment]
https://github.com/laserdisc-io/fs2-aws/runs/5852303251?check_suite_focus=true

**Review by semenodm — APPROVED** [review]

**Pull request merged**